tests/unit/amo/components/TestGetFirefoxButton.js

```diff
@@ -153,6 +158,10 @@ describe(__filename, () => {
         });
         const expectedHref = `${DOWNLOAD_FIREFOX_BASE_URL}${makeQueryStringWithUTM(
           {
+            overrideQueryParams: {
+              experimentId: EXPERIMENT_CONFIG.id,
```

Comment:
This is why I like assertions with actual strings instead of re-calling the same functions. This should be `experiment` instead of `experimentId` because we expect the query param to be name `experiment`.